Compiler fails while building version 4.1.0

Hi team!

After trying to build the latest version with Docker I discovered that something might be off with a redefinition:

```log
46.24 [INFO] -------------------------------------------------------------
46.24 [ERROR] COMPILATION ERROR : 
46.24 [INFO] -------------------------------------------------------------
46.24 [ERROR] /usr/app/src/test/java/jumper/HeaderSteps.java:[61,15] method specificRequestHeaderSet(java.lang.String,java.lang.String) is already defined in class jumper.HeaderSteps
46.24 [ERROR] /usr/app/src/test/java/jumper/EndpointSteps.java:[16,14] cannot find symbol
```

Can you please take a look when you have the time?

--- a/pom.xml
+++ b/pom.xml
@@ -13,7 +13,7 @@ SPDX-License-Identifier: Apache-2.0
 
 	<groupId>de.telekom.ei.jumper</groupId>
 	<artifactId>jumper-sse</artifactId>
-	<version>4.1.0</version>
+	<version>4.1.1</version>
 
 	<parent>
 		<groupId>org.springframework.boot</groupId>

--- a/src/test/java/jumper/HeaderSteps.java
+++ b/src/test/java/jumper/HeaderSteps.java
@@ -57,15 +57,6 @@ public class HeaderSteps {
             }));
   }
 
-  @Given("request header {word} is set to {word}")
-  public void specificRequestHeaderSet(String header, String value) {
-    baseSteps.setHttpHeadersOfRequest(
-        baseSteps.httpHeadersOfRequest.andThen(
-            httpHeaders -> {
-              httpHeaders.set(header, value);
-            }));
-  }
-
   @Given("RealRoute headers are set with x-token-exchange")
   public void realRouteHeadersSetWithXtokenExchange() {
     baseSteps.setHttpHeadersOfRequest(

